rules/S6372/cfamily/metadata.json

```
@@ -0,0 +1,19 @@
+{
+  "title": "Use type-erased \"coroutine_handle\" when applicable",
```

```suggestion
  "title": "\"await_suspend\" should accept type-erased \"coroutine_handle\" when it is used in a generic way",
```
I think a more specific rule title will help people easier judge what the rule is about when they overview a list of rules.

---

Like the suggestions. Thanks.